examples/common/chip-app-server/Server.cpp

```diff
@@ -48,8 +48,8 @@ namespace {
 class ServerCallback : public SecureSessionMgrDelegate
 {
 public:
-    void OnMessageReceived(const PacketHeader & header, Transport::PeerConnectionState * state, System::PacketBuffer * buffer,
-                           SecureSessionMgrBase * mgr) override
+    void OnMessageReceived(const PacketHeader & header, const PayloadHeader & payloadHeader, Transport::PeerConnectionState * state,
```

Comment:
Why did we add payloadHeader here?

It is not currently used by any callbacks. Lets add it when it is really needed. The call already has 4 arguments and adding a 5th unused one seems odd.

Reply:
This is the overwrite of the function OnMessageReceived defined in SecureSessionMgrDelegate. It has to change to align with the function signature update in the delegate interface.